visualization/mlchartlib/src/AccessibleChart.tsx

```diff
@@ -53,9 +53,6 @@ export class AccessibleChart extends React.Component<AccessibleChartProps, { loa
                 }
                 this.subscribeToSelections();
             }
-            if (!this.state.loading) {
```

Comment:
Why change the way loading is handled?

Reply:
It didn't have any real effect, the idea was to show something during the async part of plotly rendering, but that is a lot shorter than the syncronous part that does take a while.